docs/quick-start/developers/_deploy.md

```diff
@@ -44,25 +54,64 @@ import TabItem from '@theme/TabItem';
 
 To receive testnet ELF tokens, run this command after replacing `$WALLET_ADDRESS` and `$WALLET_PASSWORD` with your wallet details:
 
+<Tabs>
+<TabItem value="Linux and macOs" label="Linux and macOs" default>
 ```bash title="Terminal"
 export WALLET_ADDRESS="YOUR_WALLET_ADDRESS"
 curl -X POST "https://faucet.aelf.dev/api/claim?walletAddress=$WALLET_ADDRESS" -H "accept: application/json" -d ""
 ```
+</TabItem>
+
+<TabItem value="Windows" label="Windows">
+
+```bash title="Terminal"
+$headers = @{
+    "accept" = "application/json"
+}
+
+$env:WALLET_ADDRESS = "YOUR_WALLET_ADDRESS"
+
+Invoke-WebRequest -Uri "https://faucet.aelf.dev/api/claim?walletAddress=$env:WALLET_ADDRESS" -Method POST -Headers $headers -Body ""
+```
+
+</TabItem>
+</Tabs>
 
 **2. Check ELF Balance:**
 
 To check your ELF balance, use:
 
+<Tabs>
+<TabItem value="Linux and macOs" label="Linux and macOs" default>
 ```bash title="Terminal"
 aelf-command call ASh2Wt7nSEmYqnGxPPzp4pnVDU4uhj1XW9Se5VeZcX2UDdyjx -a $WALLET_ADDRESS -p $WALLET_PASSWORD -e https://tdvw-test-node.aelf.io GetBalance
 ```
+</TabItem>
+
+<TabItem value="Windows" label="Windows">
+```bash title="Terminal"
+aelf-command call ASh2Wt7nSEmYqnGxPPzp4pnVDU4uhj1XW9Se5VeZcX2UDdyjx -a $env:WALLET_ADDRESS -p $env:WALLET_PASSWORD -e https://tdvw-test-node.aelf.io GetBalance
+```
+</TabItem>
+</Tabs>
 
 You will be prompted for the following:
 
+<Tabs>
+<TabItem value="Linux and macOs" label="Linux and macOs" default>
 ```sh title="Terminal"
 Enter the required param <symbol>: ELF
 Enter the required param <owner>: **$WALLET_ADDRESS**
 ```
+</TabItem>
```

Comment:
Same for Linux, MacOS and Windows. No need for different tabs here. Check all instances